packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -285,11 +295,19 @@ export interface CallAdapterCallOperations {
     // @beta
     resumeCall(): Promise<void>;
     // @beta
+    selectUploadedBackground(backgroundImage: BackgroundImage): void;
```

Comment:
nit: would rename to `selectCustomBackground` and avoid the word uploaded

Reply:
In fact for all we should add the prefix `video`
`selectCustomVideoBackground`

Reply:
Sure, I will update by adding the prefix `video` to all the methods. But for `selectUploadedBackground` or `selectCustomVideoBackground`, this method is going to add new background to default set, select the background as active effect and then apply it if that is what design implies. Should we have a name that is bit more self explanatory?

Reply:
We'll need to wait until we get confirmation from UX design what should happen when a user uploads a custom image (does it auto apply, does it auto select and they need to hit a button called apply, or does it simply get added to the picker selection but not be selected or applied)